@msh/msh.m

```diff
@@ -1572,9 +1576,44 @@ function write(obj,fname,type)
             [p1(:,1),p1(:,2)] = m_ll2xy(p1(:,1),p1(:,2)) ;
             [p2(:,1),p2(:,2)] = m_ll2xy(p2(:,1),p2(:,2)) ;
             if nfix > 0
-                [pfixx(:,1),pfixx(:,2)] = m_ll2xy(pfixx(:,1),pfixx(:,2)) ;
+                [pfixx(:,1),pfixx(:,2)] = m_ll2xy(pfixx(:,1),pfixx(:,2));
+            end
+
+            % checking for weirs in obj1 to keep
+            if any(obj1.bd.ibtype == 24)
```

Comment:
This is problematic because a mesh may or may not have its bd structure populated. If it isn't populated this will incur a segfault.

Reply:
True. Will fix this.